src/hooks/index.js

```diff
@@ -0,0 +1,56 @@
+import { getTodoList } from "../api";
+import { useQuery } from "@tanstack/react-query";
+import { useQueryClient, useMutation } from "@tanstack/react-query";
+import { deleteTodo } from "../api";
+import { updateTodo } from "../api";
+
+export const useGetTodoList = (options = {}) => {
+  return useQuery({
+    queryKey: ["todo-list"],
+    queryFn: getTodoList,
+    ...options,
+  });
+};
+
+export const useAddTodo = () => {
+  const queryClient = useQueryClient();
+  return useMutation({
```

Comment:
what is the purpose of this function, the name is `useAddTodo` but you pass `updateStatus` function to `mutationFn`

Reply:
Done.